src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Note>
+    This is an experimental feature.
+</Note>
+
+
+You can monitor the performance of your views in a SwiftUI project with SentrySwiftUI library.
+
+Install it with CocoaPod:
+
+```ruby
+platform :ios, '13.0'
+use_frameworks! # This is important
+
+target 'YourApp' do
+  pod 'SentrySwiftUI'
+end
+```
+
+Or by importing sentry-cocoa repository (https://github.com/getsentry/sentry-cocoa) with SPM.
+
+
+In order to use it, you need to wrap your view with `SentryTracerView`, like this:
+```swift {tabTitle:Swift}
+
+import SentrySwiftUI
+
+var body: some View {
+    SentryTracedView("<NAME YOU TRANSACTION>"){
```

Comment:
`m`: It's not only the name of the transaction, it's also the name of a span if you nest them.
```suggestion
    SentryTracedView("<NAME YOUR TRANSACTION>"){
```